inference-engine/tests/functional/plugin/cpu/shared_tests_instances/execution_graph_tests/num_inputs_fusing_bin_conv.cpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "execution_graph_tests/num_inputs_fusing_bin_conv.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+INSTANTIATE_TEST_CASE_P(inputsNumFusingBinConv, ExecGraphOneInputFusingBinConv,
+                        ::testing::Values(fusingBinConvParams{{ 1, 16, 30, 30}, {2, 2}, {3, 3}, 16, 4, CommonTestUtils::DEVICE_CPU}),
```

Comment:
Only Depthwise convolutions can be fused into BinConv, It looks like you are testing just grouped conv.